### PR TITLE
Fix minor typos on model documentation

### PR DESCRIPTION
**Pull request by tarpier**
#### Description of what you did:

found 3 minor typos and replaced them

### CODE REVIEW

File: docs/3.0.0-beta.x/concepts/models.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ To improve the Developer Experience when developing or using the administration
 
 ### Exceptions
 
-- `uid` — This field type allows a `targetField` key. The value is the name of an attribute thas has `string` of `text` type.
+- `uid` — This field type allows a `targetField` key. The value is the name of an attribute that has `string` of `text` type.
```
Comment (derrickmehaffy) [inline docs/3.0.0-beta.x/concepts/models.md:221]: Can you also please include a fix to the `v3.x` part of the docs for this? Thank you

Comment (tarpier) [reply docs/3.0.0-beta.x/concepts/models.md:221]: 2 were already fixed, 3rd is now on 3.x branch also

**Review by derrickmehaffy — APPROVED** [review]
LGTM

**Comment by lauriejim** [comment]
Hello, can you please validate the DCO - thanks.
Then I will be able to merge your PR.
Thank you for this fix.

**Pull request merged**